kuma/javascript/src/payments/components/errors.jsx

```diff
@@ -0,0 +1,49 @@
+// @flow
+import * as React from 'react';
+import { gettext, Interpolated } from '../../l10n.js';
+
+type Props = {
+    text: string,
+    onClick?: (event: SyntheticEvent<HTMLAnchorElement>) => void,
```

Comment:
Looks like all the instances use the retry button atm, so I'd go with making `onClick` a required prop